### PR TITLE
refactor: update hyper-parameters for first-order algorithms

### PR DESCRIPTION
**Pull request by muchvo**
## Description

1. Update hyper-parameters for first-order algorithms.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Implemented Tasks

- [ ] Subtask 1
- [ ] Subtask 2
- [ ] Subtask 3

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://omnisafe.readthedocs.io/en/latest/developer/contributing.html) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.
- [x] I have reformatted the code using `make format`. (**required**)
- [x] I have checked the code using `make lint`. (**required**)
- [x] I have ensured `make test` pass. (**required**)

### CODE REVIEW

**Review by zmsn-2077 — APPROVED** [review]
LGTM.

File: benchmarks/README.md [hunk 1]
```diff
@@ -23,12 +23,12 @@ The OmniSafe Mujoco Velocity Benchmark assesses the efficacy of OmniSafe's SafeR
 - **[ICML 2017]** [Proximal Constrained Policy Optimization (PCPO)](https://proceedings.mlr.press/v70/achiam17a)
 - **[ICLR 2019]** [Reward Constrained Policy Optimization (RCPO)](https://openreview.net/forum?id=SkfrvsA9FX)
 
-> **More details can be refer to [On Policy Experiment](https://github.com/PKU-MARL/omnisafe/tree/main/benchmarks/on-policy/README.md).**
```
Comment (Gaiejj) [inline benchmarks/README.md:26]: What about using remote link ?

File: benchmarks/README.md [hunk 2]
```diff
@@ -23,12 +23,12 @@ The OmniSafe Mujoco Velocity Benchmark assesses the efficacy of OmniSafe's SafeR
...
+> **More details can be refer to [On Policy Experiment](./on-policy/README.md).**
 
 ## Off-Policy
 ### Supported Algorithms
 - [Deep Deterministic Policy Gradient (DDPG)](https://arxiv.org/pdf/1509.02971.pdf)
 - [Twin Delayed DDPG (TD3)](https://arxiv.org/pdf/1802.09477.pdf)
 - [Soft Actor-Critic (SAC)](https://arxiv.org/pdf/1812.05905.pdf)
 
-> **More details can be refer to [Off Policy Experiment](https://github.com/PKU-MARL/omnisafe/tree/main/benchmarks/off-policy/README.md).**
```
Comment (Gaiejj) [inline benchmarks/README.md:34]: The same

**Review by Gaiejj — CHANGES_REQUESTED** [review]

**Review by Gaiejj — APPROVED** [review]
LGTM.

**Pull request merged**